### ISSUE TITLE
Create issue on feedstock if PR could not be created

### ISSUE THREAD

**Issue by jaimergp** [regro/cf-scripts#1534]
Sometimes the bot will fail to post the PR even if a new version was detected. These warnings are posted to https://conda-forge.org/status/#version_updates, but I am not sure people actively check those out.

We could think of having the bot post an issue on the feedstock so the maintainers are aware of the error (in very conservative ways, trying to reduce spam!)

**Comment by leofang** [comment]
It would be nice indeed. Usually when the bot fails to send an update, there are issues in the recipe that feedstock maintainers can help fix.

**Comment by h-vetinari** [comment]
Yeah, I remember suggesting this to Matthew a long time ago (cannot find it anymore where), great to have an issue for this!

Key point is not to spam feedstocks with issues, i.e. I think we should use a particular formatting / magic cookie to let the bot detect whether a "version upgrade failed" issue already exists, and only open an issue if it doesn't exist yet.

**Comment by jaimergp** [comment]
This GH Action has "edit existing issue instead of creating new one" built-in: https://github.com/JasonEtco/create-an-issue

**Comment by beckermr** [comment]
Yep. We'd likely code that logic ourselves in python. It exists in other places in conda-forge infrastructure so that'd be straightforward.
